rfs_test/TEST_assembly_schema.py

```diff
@@ -34,4 +34,84 @@
 
 '''
 
-## end Assertion_ASSE114
\ No newline at end of file
+
+def Assertion_ASSE114(self, log):
+
+    log.AssertionID = 'ASSE114'
+    assertion_status = log.PASS
+    log.assertion_log('BEGIN_ASSERTION', None)
+
+    authorization = 'on'
+    rq_headers = self.request_headers()
+
+    json_payload, headers, status = self.http_GET(
+        '/redfish/v1/Assembly', rq_headers, authorization)
```

Comment:
Will need to look for an Assembly resource; it will likely not be found at /redfish/v1/Assembly.

Best place to find this would be on a given Chassis resource:
- From Service Root, find the Chassis Collection
- In the Chassis Collection, pick a Chassis instance
- In the Chassis instance, look for the Assembly link
  - If not found, skip the test